ts/components/messages/MessageDetailRawInfoComponent.tsx

```diff
@@ -90,13 +87,11 @@ class MessageDetailRawInfoComponent extends React.PureComponent<Props, State> {
                   )}: `}</Text>
                   {service.department_name}
                 </Text>
-
-                <Text link={true} onPress={onServiceLinkPress}>
-                  <Text bold={true}>{`${I18n.t(
-                    "messageDetails.rawInfoLabels.serviceName"
-                  )}: `}</Text>
-                  {service.service_name}
-                </Text>
+                <TouchableDefaultOpacity onPress={onServiceLinkPress}>
+                  <Text link={true}>
```

Comment:
```suggestion
                  <Text bold={true}>
```
futhermore you completly remove this 
`{service.service_name}`

| PRE  | THIS PR |
| ------------- | ------------- |
![pre](https://user-images.githubusercontent.com/822471/68791617-f4f07500-0649-11ea-991e-1bc0305ebc5f.png) | ![Simulator Screen Shot - iPhone X - 2019-11-13 at 19 13 21](https://user-images.githubusercontent.com/822471/68791614-f457de80-0649-11ea-9cb4-55a7796d23ca.png)

Reply:
Ok, I solved it but I must be use bold for "Servizio" and link for "service.service_name".